### ISSUE TITLE
'`$NUMBER` new items' button has insufficient focus indication

### ISSUE THREAD

**Issue by lee0c** [hometown-fork/hometown#1380]
### Steps to reproduce the problem

1. With auto-load turned off, tab to the button at the top of the feed that loads new posts
2. Notice that only the bottom border has a focus outline

### Expected behaviour

Fully surrounding the button with a focus indicator

### Actual behaviour

Only bottom border shows focus

### Detailed description

I would recommend reading [A guide to designing accessible focus indicators](https://www.sarasoueidan.com/blog/focus-indicators/) for more information about why incomplete borders can lack required focus indication.

### Mastodon instance

flipping.rocks

### Mastodon version

v4.2.17+hometown-1.1.2

### Browser name and version

Firefox 145.0.2

### Operating system

Win11

### Technical details

This can be adjusted with margin, padding, or a small negative `outline-offset`

**Comment by lee0c** [comment]
I can help with this.

**Comment by nachtjasmin** [comment]
<img width="613" height="120" alt="Screenshot of the 'load more' button, focused with a clear outline around the whole button." src="https://github.com/user-attachments/assets/50d377b8-9852-4e93-acd9-d5a72bc34e80" />

Issue not longer present in current versions, tested with `v4.5.11+hometown-1.2.1`.
